lib/Core/PTree.cpp

```diff
@@ -9,38 +9,39 @@
 
 #include "PTree.h"
 
+#include "klee/ExecutionState.h"
 #include "klee/Expr/Expr.h"
 #include "klee/Expr/ExprPPrinter.h"
 
 #include <vector>
 
 using namespace klee;
 
-PTree::PTree(const data_type &root) : root(new Node(nullptr, root)) {}
+PTree::PTree(ExecutionState *initialState) {
+  root = std::make_unique<PTreeNode>(nullptr, initialState);
+  initialState->ptreeNode = root.get();
```

Comment:
I think this is redundant with your `PTreeNode(PTreeNode *parent, ExecutionState *state)` as defined below

Reply:
Good catch, thanks!